index.html

```diff
@@ -602,6 +602,9 @@ <h2 slot=header>
           <a href=/examples/input-output.html>&lt;input-output&gt;</a>
 
         <li>
+          <a href=/examples/to-do>&lt;to-do&gt;</a>
```

Comment:
@tmornini there's TONS of them actually. Been in the spec (even IE6) since like 2000. DOM Level 2.

Surprised you didn't say anything about `<html>` the `<body>` tag and the `<head>` tag... or lack thereof ;-) 
https://www.w3.org/TR/html5/syntax.html#optional-tags